src/components/App/App.js

```diff
@@ -37,14 +42,10 @@ class App extends React.Component {
       .catch((err) => console.error(err));
   }
 
-  componentDidUpdate() {
-    console.log(this.state.areas);
-  }
-
   render() {
     return (
       <div className="App">
-        <Header userName={this.state.user.name} />
+        <Header user={this.state.user}  />
```

Comment:
Can you explain why this change was possible? Is the user empty string value in state set to the user object created on sign in?